lib/linguist/languages.yml

```diff
@@ -53,6 +53,14 @@ ABAP:
   - ".abap"
   ace_mode: abap
   language_id: 1
+ABAP CDS:
```

Comment:
Should this be grouped (`group:`) under ABAP? The consequence is that it would be displayed as ABAP in language statistics.

Reply:
No, ABAP and ABAP CDS are completely different languages.